src/macros.php

```diff
@@ -52,6 +52,29 @@
     });
 }
 
+if (! CrudColumn::hasMacro('linkTo')) {
+    CrudColumn::macro('linkTo', function ($route, $target = null) {
+        $wrapper = $this->attributes['wrapper'] ?? [];
+        if (in_array($this->attributes['type'], ['select', 'select_grouped', 'select2', 'select2_grouped', 'select2_nested', 'select2_from_ajax'])) {
+            $wrapper['href'] = function ($crud, $column, $entry, $related_key) use ($route) {
+                return route($route, $related_key);
+            };
+        } else {
+            $wrapper['href'] = function ($crud, $column, $entry) use ($route) {
+                return url($route.$column['value']);
```

Comment:
This is throwing errors all over the place. I can't really see how this would work @karandatwani92 . Why do we use `route()` above, and `url()` here?

Is this a typo? Did you mean to do something like this?

```suggestion
                return route($route, $column['value']);
```

Reply:
Oh wait. This is in order to cover the `'https://www.google.com/search?q='` scenario, right? I don't remember us deciding to do that. It's tricky for the same function to support both, because 
- both strings `category.show` and `https://www.google.com/search?q=` are strings;
- using full URLs like `https://www.google.com/search?q=` is unlikely, most likely if people need to point to another link it would be a Laravel route or URL;
- even if you switched to support both `route()` and `url()` the system can't reliably differentiate between `category.show` and `category/{id}/show`;

So I don't think we should support both URL and ROUTE here. For all intended purposes, this function should be `linkTo($routeName)`. For anything more custom, they can customize the wrapper attributes manually.